packages/block-editor/src/components/inserter/style.scss

```diff
@@ -88,6 +88,10 @@ $block-inserter-tabs-height: 44px;
 			background: $white;
 			box-shadow: 0 0 0 $border-width-focus $theme-color;
 		}
+
+		&::placeholder {
+			color: #6c6f75;
```

Comment:
Can you use `$dark-gray-400` instead? That's #606a73.

Reply:
Thank you @jasmussen! $dark-gray-400 gets us a 5.01 passing score. Much better.